### ISSUE TITLE
Spilo and postgres-operator compatibility

### ISSUE THREAD

**Issue by hedgss** [zalando/postgres-operator#666]
Hi all,

I see that new versions of spilo are available, but I don't know if the postgres-operator supports it.

Could you please clarify what spilo versions are supported by the postgres-operator.?
Do you have some compatibility matrix os something similar?

**Comment by FxKu** [comment]
A feature matrix (or at least a mentioning in the release notes) is a good idea. Both projects are tightly coupled and new versions of Spilo will not break things for the operator.

**Comment by hedgss** [comment]
as far as I understand postgres-operator can manage spilo version that is specified in manifests or newer. Is it correct for all postgres-operator versions?

**Comment by Jan-M** [comment]
Yes, using a newer Spilo should always be fine. Not immediately aware of breaking changes in how it is configured.
